src/fosslight_util/oss_item.py

```diff
@@ -10,109 +10,171 @@
 
 
 class OssItem:
-    name = "-"
-    version = ""
-    licenses = []
-    source = ""
-    files = []
-    copyright = ""
-    comment = ""
-    exclude = ""
-    homepage = ""
-    relative_path = ""
-
     def __init__(self, value):
-        self.name = "-"
-        self.version = ""
-        self.licenses = []
+        self._name = "-"
+        self._version = ""
+        self._licenses = []
         self.source = ""
-        self.files = []
-        self.copyright = ""
+        self._files = []
+        self._copyright = ""
         self.comment = ""
-        self.exclude = ""
+        self._exclude = False
         self.homepage = ""
         self.relative_path = value
+        self._source_name_or_path = []
+        self.download_location = ""
+        self._copyright_text = ""
 
     def __del__(self):
         pass
 
-    def set_homepage(self, value):
-        self.homepage = value
+    @property
+    def copyright(self):
+        return self._copyright
+
+    @copyright.setter
+    def copyright(self, value):
+        if value != "":
+            if isinstance(value, list):
+                value = "\n".join(value)
+            value = value.strip()
+        self._copyright = value
 
-    def set_comment(self, value):
-        self.comment = value
+    @property
+    def copyright_text(self):
+        return self._copyright_text
 
-    def set_copyright(self, value):
+    @copyright_text.setter
+    def copyright_text(self, value):
         if value != "":
             if isinstance(value, list):
                 value = "\n".join(value)
             value = value.strip()
-        self.copyright = value
+        self._copyright_text = value
```

Comment:
It seems that copyright_text and copyright are overlapping variables.

Reply:
Remove the duplicated copyright_text